src/styles/index.ts

```diff
@@ -432,6 +432,12 @@ const styles = (theme: ThemeColors) =>
             lineHeight: variables.lineHeightHero,
         },
 
+        textHeroSmall: {
```

Comment:
Can you share a screenshot of where this is used exactly?

Reply:
How much different is this from our standard h1 styles? I would love to make sure our font styles don't get out of control, so if we have an existing style to use, let's use it. cc @Expensify/design

Reply:
@shawnborton  I looked into it and yes, it doesn't really make sense here to create a new style. I've replaced all use cases of `textHeroSmall` with a combination of existing styles `[styles.textHeadlineH1, styles.textXXLarge]`. It looks the same as before but without needing new styles.